Automattic/pocket-casts-ios
# Fix 1686 grid margins go off screen

Fixes #1686 

This PR updates the search bar code and mini-player code to ensure it does not change any existing horizontal insets when updating the insets.

## To test

 - Start the app
 - On podcasts view pull to refresh, and while the pull to refresh is ongoing pull to refresh again.
 - Check that the horizontal margins on the grid layout are kept
 - Start the playing any podcast to see the mini-player
 - Check if you have a bottom margin at the end of scroll the podcasts view
 - Tap on the mini-player artwork and choose `Close and Clear Up Next`
 - Check that the bottom at the end of the scroll is still there and horizontal margins are kept

## Checklist
